### PR TITLE
drupal_tag => tag

### PR DESCRIPTION
**Pull request by ashleypt**
I'm back :D

Part of #956

it is failing one test, as you should see in travis. @jywarren pointers?

* [x] all tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request is descriptively named with #number reference back to original issue
* [x] if possible, multiple commits squashed if they're smaller changes
* [x] reviewed/confirmed/tested by another contributor or maintainer
* [x] `schema.rb.example` has been updated if any database migrations were added

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -343,5 +343,8 @@ DEPENDENCIES
   will_paginate (>= 3.0.6)
   will_paginate-bootstrap (>= 1.0.1)
 
+RUBY VERSION
```
Comment (jywarren) [inline Gemfile.lock]: Hi, welcome back! Great to see your work here. 

I'm curious, do you have Gemfile.lock in your .gitignore file? I was wondering why its been added to your changes; just checking! We don't need it to be, I believe. 

I'll go through more of this soon, thank you!

**Comment by jywarren** [comment]
Hmm, it seems to be passing, and wow, a very thorough body of work! It's rebased on top of the latest master?

We now have a staging server we can try things like this out on before going live with them. So I have more confidence in merging this. What do you think?

**Comment by ashleypt** [comment]
go ahead and try it on the server, sure :)

If it passes on Travis and on the staging server, it must have been either
on my end or specific to sqlite.

As for my Gemfile.lock, I haven't edited the .gitignore

**Comment by jywarren** [comment]
Great, can you revert the changes from the Gemfile.lock and push up one more commit? Thanks!!

**Comment by ashleypt** [comment]
@jywarren should be all better now :)

I'll go through the rest of those #956 issues too maybe :D

**Comment by jywarren** [comment]
Merged!!! I'm going to move it to the stable branch so it can be tested out there before publishing. Thank you, amazing work!

**Comment by jywarren** [comment]
OK - it's building here, now: http://staging.laboratoriopublico.org/

Once it's done, it can be tested out here; http://testing.laboratoriopublico.org/

Give me a thumbs up after making/editing/deleting a few tags and checking out the tags pages?

Thank you!!! 🎉

**Comment by ashleypt** [comment]
i will tomorrow sure :)

ill need to make an account and all that first but I'll let you know

**Pull request merged**